nixos/lib/testing.nix

```diff
@@ -1,6 +1,6 @@
-{ system, minimal ? false, config ? {} }:
+{ system, minimal ? false, config ? {}, pkgs ? import ../.. { inherit system config; }, extraConfigurations ? [] }:
```

Comment:
`? import ../.. { inherit system config; },`

We have the same default in the other file already, so I’d say one should go.

Reply:
It would break backward compatibility.